src/pages/projectSettings/tabs/PeriodicBuildsTab/getFormSchema.ts

```diff
@@ -0,0 +1,126 @@
+import { Field } from "@rjsf/core";
+import { SpruceFormProps } from "components/SpruceForm";
+import widgets from "components/SpruceForm/Widgets";
+import { timeZones } from "constants/fieldMaps";
+import { form, ProjectType } from "../utils";
+
+const { overrideRadioBox } = form;
+
+export const getFormSchema = (
+  projectType: ProjectType,
+  timezone: string
+): {
+  fields: Record<string, Field>;
+  schema: SpruceFormProps["schema"];
+  uiSchema: SpruceFormProps["uiSchema"];
+} => ({
+  fields: {},
+  schema: {
+    type: "object" as "object",
+    description: "Configure tasks to run at fixed intervals.",
+    ...overrideRadioBox(
+      "periodicBuilds",
+      ["Override Repo Commands", "Default to Repo Commands"],
+      {
+        type: "array" as "array",
+        default: [],
+        items: {
+          type: "object" as "object",
+          properties: {
+            intervalHours: {
+              type: "number" as "number",
```

Comment:
I don't know if this is relevant to your PR but if I type a non-integer into the text box it shows the error, and then when I press the back button it just takes away the error but doesn't actually delete the character.
<img width="496" alt="image" src="https://user-images.githubusercontent.com/26798134/161625425-475e0271-d537-465a-b481-8a8cfc038d10.png">

Reply:
Not related to these changes but it is being tracked! EVG-16641